projects/laji/src/i18n/fi.json

```diff
@@ -439,7 +438,6 @@
   "friend.requests": "Kaveripyynnöt",
   "friend.requestSent": "Kaveripyyntö lähetetty",
   "gathering.section.outsideSection": "Ei osiota",
-  "geoConvert.error.too_complex": "Joidenkin havaintojen geometriat ovat liian monimutkaisia .shp-tiedostomuodolle (GeometryCollection). Käytä sen sijaan .gpkg-muotoa tai valitse toinen geometriatyyppi.",
```

Comment:
Did you delete references of "too_complex"?

Reply:
I had forgotten to delete it, deleted it now